kyaml/kio/filters/container.go

```diff
@@ -33,6 +33,9 @@ type ContainerFilter struct {
 	// Network is the container network to use.
 	Network string `yaml:"network,omitempty"`
 
+	// LocalVolume is the volume the container uses.
+	LocalVolume string `yaml:"localVolume,omitempty"`
```

Comment:
We should probably make this a struct so the type and dst can be specified as well.